changelogs/unreleased/gh-7523-sigsegv-on-m1-macs.md

```diff
@@ -1,4 +1,4 @@
 ## bugfix/core
```

Comment:
> NO_TEST=Hard to create a stable reproducer

Maybe I'm naïve, but why wouldn't a unit test, in which you, in one thread, spam some signal and call `backtrace_collect` in another thread?

Reply:
Okay, thanks for elaborating!

Reply:
Nit: next time, please add such comments to the NO_TEST tag, so that no questions arise.